core/src/main/java/io/jenkins/pluginhealth/scoring/scores/SecurityWarningScoring.java

```diff
@@ -41,31 +41,35 @@ public class SecurityWarningScoring extends Scoring {
 
     @Override
     public List<ScoringComponent> getComponents() {
-        return List.of(
-            new ScoringComponent() {
-                @Override
-                public String getDescription() {
-                    return "The plugin must not have on-going security advisory.";
-                }
+        return List.of(new ScoringComponent() {
+            @Override
+            public String getDescription() {
+                return "The plugin must not have on-going security advisory.";
+            }
 
-                @Override
-                public ScoringComponentResult getScore(Plugin $, Map<String, ProbeResult> probeResults) {
-                    final ProbeResult probeResult = probeResults.get(KnownSecurityVulnerabilityProbe.KEY);
-                    if (probeResult == null || ProbeResult.Status.ERROR.equals(probeResult.status())) {
-                        return new ScoringComponentResult(-100, 100, List.of("Cannot determine if plugin has on-going security advisory."));
-                    }
-                    if ("No known security vulnerabilities.".equals(probeResult.message())) {
-                        return new ScoringComponentResult(100, getWeight(), List.of("Plugin does not seem to have on-going security advisory."));
-                    }
-                    return new ScoringComponentResult(0, getWeight(), List.of("Plugin seem to have on-going security advisory.", probeResult.message()));
+            @Override
+            public ScoringComponentResult getScore(Plugin $, Map<String, ProbeResult> probeResults) {
+                final ProbeResult probeResult = probeResults.get(KnownSecurityVulnerabilityProbe.KEY);
+                if (probeResult == null || ProbeResult.Status.ERROR.equals(probeResult.status())) {
+                    return new ScoringComponentResult(
+                            -100, 100, List.of("Cannot determine if plugin has on-going security advisory."));
                 }
-
-                @Override
-                public int getWeight() {
-                    return 1;
+                if ("No known security vulnerabilities.".equals(probeResult.message())) {
+                    return new ScoringComponentResult(
+                            100, getWeight(), List.of("Plugin does not seem to have on-going security advisory."));
                 }
+                return new ScoringComponentResult(
+                        0,
+                        getWeight(),
+                        List.of("Plugin seem to have on-going security advisory.", probeResult.message()),
+                        List.of(new Resolution(probeResult.message())));
```

Comment:
No. This is not correct.
See the resolution description to see how to use it.

Reply:
update the resolution parameter text:url
```
  return new ScoringComponentResult(
                        0,
                        getWeight(),
                        List.of("Plugin seem to have on-going security advisory.", probeResult.message()),
                        List.of(new Resolution(probeResult.message().substring(0, index), probeResult.message().substring(index))));
```